Lib/inspect.py

```diff
@@ -454,7 +459,7 @@ def isgenerator(object):
         gi_frame        frame object or possibly None once the generator has
                         been exhausted
         gi_running      set to 1 when generator is executing, 0 otherwise
-        next            return the next item from the container
+        gi_yieldfrom    is object being iterater by yield from?
```

Comment:
This can be None according to inspect.rst (maybe use the same wordings actually?)